login-fire-common-behavior.html

```diff
@@ -260,7 +262,12 @@
       _user: {
         type: Object,
         observer: '_userChanged'
-      }
+      },
+
+      _showOneButton: {
```

Comment:
It seems related to the `fire-login-button`. Its value is `false` only when we can't signing out and we are already signed in. What's the benefit to have this property?